release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -193,6 +193,24 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-8-bug-fixes"]
 == Bug fixes
 
+*Bare Metal Hardware Provisioning*
+
+* Previously, Ironic failed to download an image for installation because it uses HTTPS by default and Ironic does not have the correct certificate bundle available. This issue is fixed by setting the image download as insecure to request a transfer without the certificate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1953795[*BZ#1953795*])
```

Comment:
Restate the subject--it gets restated after "it" and that _could_ cause confusion.
s/it/Ironic/

Continue with the past tense that is already in use:
s/does not/did not/